easybuild/easyblocks/m/mathematica.py

```diff
@@ -76,16 +76,17 @@ def install_step(self):
             cmd = self.cfg['preinstallopts'] + './' + install_script
             shortver = '.'.join(self.version.split('.')[:2])
             qa_install_path = os.path.join('/usr', 'local', 'Wolfram', self.name, shortver)
-            qa = {
-                r"Enter the installation directory, or press ENTER to select %s: >" % qa_install_path: self.installdir,
-                r"Create directory (y/n)? >": 'y',
-                r"Should the installer attempt to make this change (y/n)? >": 'n',
-                r"or press ENTER to select /usr/local/bin: >": os.path.join(self.installdir, "bin"),
-            }
+            qa = [
+                (r"Enter the installation directory, or press ENTER to select %s: >" % qa_install_path,
+                 self.installdir),
+                (r"Create directory \(y/n\)\? >", 'y'),
+                (r"Should the installer attempt to make this change \(y/n\)\? >", 'n'),
+                (r"or press ENTER to select /usr/local/bin: >", os.path.join(self.installdir, "bin")),
```

Comment:
There are a couple of line breaks in this question, so we need to use `[\s\n]*`:
```suggestion
               (r"or press ENTER to select[\s\n]*/usr/local/.*:[\s\n]*>", os.path.join(self.installdir, "bin")),
```